dnf-docker-test/features/steps/test_behave.py

```diff
@@ -64,6 +65,14 @@ def diff_package_lists(a, b):
         map(_right_decorator, list(sb - sa)))
 
 
+def diff_query_lists(sack_a, sack_b):
```

Comment:
Sack is loaded only with system repo. So far there is not need to load available packages.